docs/modules/ROOT/pages/explanations/decisions/deployments.adoc

```diff
@@ -0,0 +1,66 @@
+= Decision How to Deploy Compositions and XRDs
+
+== Problem
+
+We need to deploy our XRDs and Compositions for AppCat to OpenShift and Kubernetes clusters.
+Also writing the compositions and XRDs plainly is somewhat a hassle (yaml engineering).
+Having helpers with engineering compositions and XRDs would be a welcomed bonus for the solution.
```

Comment:
```suggestion
Having helpers with engineering Compositions and XRDs would be a welcomed bonus for the solution.
```

Please be consistent with capitalization throughout the document, that's just one occurence, I'll not fix the others.